amy/workshops/lookups.py

```diff
@@ -222,6 +222,31 @@ def get_queryset(self):
         return results
 
 
+
```

Comment:
There should be two empty lines before class definition, three is PEP8 violation. Similar situation at the end of this hunk.

Reply:
@pbanaszkiewicz So I notice black changes are cluttering the PR a bit. Might be nice to have a single larger pr to handle flake8/black formatting. But no preference as long as it's looks readable to you.

Reply:
@lauryndbrown you're right. https://github.com/carpentries/amy/issues/1812